source/Calamari.Tests/KubernetesFixtures/Integration/HelmCliTests.cs

```diff
@@ -77,7 +82,6 @@ public void UsesCustomHelmExecutableFromPackage()
 
             var (helm, commandLineRunner, workingDirectory, _) = GetHelmCli(expectedExecutable, new CalamariVariables
             {
-                [SpecialVariables.Helm.Packages.CustomHelmExePackageKey] = expectedPackageKey,
```

Comment:
I couldn't find where this variable was being used. The current tests pass even without this here.